industry_restaurant/data/res_config_settings.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<odoo noupdate="1">
+    <record id="res_config_settings_enable_restaurant" model="res.config.settings">
+        <field name="group_uom" eval="1"/>
```

Comment:
you enable UOM only for "product_product_44" which is not really used. Maybe overkill and we don't really need UOM and/or the product 44